packages/devtools/src/hooks/useOperatingSystem.ts

```diff
@@ -0,0 +1,12 @@
+import { useMemo } from 'react';
+
+const useOperatingSystem = () => {
+  const isMac = useMemo(
+    () => /mac/i.test((navigator as any).userAgentData?.platform || navigator.userAgent),
```

Comment:
NIT: IMO this can just be a local variable. Not really needed as a hook. A user's OS won't really change right?

Reply:
Ah okay, FWIW, i meant it can just be a variable. No need to be in react world since it'll get evaluated on script-load